templates/ib/scenes-swift4.stencil

```diff
@@ -81,6 +81,21 @@ import {{module}}
     }
     return controller
   }
+
+  @available({% if isAppKit %}OSX 10.15{% else %}iOS 13.0, tvOS 13.0{% endif %}, *)
+  {{accessModifier}} func instantiate(creator block: @escaping (NSCoder) -> T?) -> T {
+    let identifier = {% if isAppKit %}NSStoryboard.SceneIdentifier({% endif %}self.identifier{% if isAppKit %}){% endif %}
+    {% if isAppKit %}
+    guard let controller = storyboard.storyboard
+      .__instantiate{{controller}}(withIdentifier: identifier, creator: block) as? T else {
```

Comment:
Why the `__` for AppKit? From what I can see in the docs, the method has no `__`.